### PR TITLE
Feature: Create client interface for manual insertion of parameter sets

### PR DESCRIPTION
**Pull request by tsirif**
See commit messages for description.

### CODE REVIEW

File: src/metaopt/client/manual.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    :param raise_exc: whether an inappropriate tuple of parameters will raise
+       an exception or it will be ignored
+
+    .. info:: If `raise_exc` is True, no set of parameters will be inserted. If
+       it is False, only the valid ones will be inserted; the rest will be ignored.
+
+    .. note:: This cannot be used to prepopulate a future experiment. So,
+       an experiment with `experiment_name` should already be configured in
+       the database.
+
```
Comment (bouthilx) [inline src/metaopt/client/manual.py]: It would be nice to have an example here.

**Comment by bouthilx** [comment]
@tsirif Ping :)

**Comment by bouthilx** [comment]
@tsirif Needs rebase

**Comment by tsirif** [comment]
@bouthilx What should we do with this? rebase and merge it?

**Comment by bouthilx** [comment]
Needs flake8 fix. Then rebase and can be merged yes.

**Comment by bouthilx** [comment]
@tsirif Many parts of this commit have been integrated in PR #70. It would be better to wait for #70 to be merged and then take care of this one. I can take care of the rebase since I'm reviewing #70 and I'll ask you to review the rebase to make sure nothing important dissapeared. The rebase will be quite different and smaller than this PR.

**Comment by bouthilx** [comment]
Rebased in #79

**Pull request closed without merging**